lib/foreman_host_reports/engine.rb

```diff
@@ -18,7 +18,7 @@ class Engine < ::Rails::Engine
 
     initializer 'foreman_host_reports.register_plugin', :before => :finisher_hook do |_app|
       Foreman::Plugin.register :foreman_host_reports do
-        requires_foreman '>= 3.0.0'
+        requires_foreman '>= 3.2.0'
```

Comment:
@lzap, I guess you need to know about that :/ @Ron-Lavi, I guess we're not going to backport this to one of the previous versions of Foreman?

Reply:
I guess we are not going to backport it in core,
but I guess this plugin will be official from 3.2? cc @lzap